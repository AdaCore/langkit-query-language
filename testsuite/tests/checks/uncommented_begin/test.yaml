driver: 'checker'
rule_name: Uncommented_BEGIN_In_Package_Bodies
project: 'prj.gpr'
