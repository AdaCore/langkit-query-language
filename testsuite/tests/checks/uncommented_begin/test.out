uncommented.adb:15:1: rule violation: mark BEGIN with package name
15 | begin                 --  FLAG
   | ^^^^^

